src/components/auth/Login.jsx

```diff
@@ -1,15 +1,15 @@
 import { useEffect, useState } from 'react';
```

Comment:
Just making a note that this component will need loading state but I believe that may be a ticket so that we can make a sweep of the code base and add where needed. But please add as you go along if you are able to.